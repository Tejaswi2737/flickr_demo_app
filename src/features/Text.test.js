import React from 'react';
import renderer from 'react-test-renderer';
import {
    DefaultText,
    RegularText,
    CustomizedText,
} from './Text.stories';

it('renders correctly', () => {
    const tree = renderer.create(<DefaultText {...DefaultText.args} />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});

it('renders correctly', () => {
    const tree = renderer.create(<RegularText {...RegularText.args} />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});

it('renders correctly', () => {
    const tree = renderer.create(<CustomizedText {...CustomizedText.args} />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
