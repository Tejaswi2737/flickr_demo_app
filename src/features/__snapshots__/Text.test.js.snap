// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdvvtL eiyFLD"
  color="darkest"
  height="medium"
  size="medium"
>
  Hello
</div>
`;

exports[`renders correctly 2`] = `
<div
  className="sc-bdvvtL bcCCPR"
  color="dark"
  height="medium"
  size="medium"
>
  Hello
</div>
`;

exports[`renders correctly 3`] = `
<div
  className="sc-bdvvtL liSZsL"
  color="dark"
  height="medium"
  size="small"
>
  Hello
</div>
`;
